lib/Documentable/Index.pm6

```diff
@@ -13,18 +13,17 @@ method new(
     :$meta!,
     :$origin!
 ) {
-
-    my $name;
-    if $meta.elems > 1 {
-        my $last = textify-pod $meta[*-1];
-        my $rest = $meta[0..*-2];
-        $name = "$last ($rest)";
+    my ($name, $category);
+    if $meta.elems == 2 {
+        $category = $meta[0];
+        $name = $meta[1];
     } else {
-        $name = textify-pod $meta;
+        warn "At $origin.url() $meta.raku() is not formatted properly";
     }
 
     nextwith(
         kind     => Kind::Reference,
+        categories => [$category.trim],
```

Comment:
Categories are not trimmed in origin?

Reply:
Not necessary. When creating an Index object, we get raw `pod.meta` of a `X<>`, so if the user types `X< foo, bar>` which is technically correct, both the category and the term will have a space prefix.
Looking back at it, I think the case where the name should be trimmed can be even more common, so I added it to the name too.
And in case the format is wrong, the exception is prevented with `.?` (the condition has already warned about itself, no need to see another exception).